modules/apps/29-fee/types/expected_keepers.go

```diff
@@ -14,12 +14,16 @@ type AccountKeeper interface {
 	GetAccount(sdk.Context, sdk.AccAddress) types.AccountI
 }
 
+// ICS4Wrapper defines the expected ICS4Wrapper for middleware
+type ICS4Wrapper interface {
+	SendPacket(ctx sdk.Context, channelCap *capabilitytypes.Capability, packet ibcexported.PacketI) error
+}
+
 // ChannelKeeper defines the expected IBC channel keeper
 type ChannelKeeper interface {
+	SendPacket(ctx sdk.Context, channelCap *capabilitytypes.Capability, packet ibcexported.PacketI) error
```

Comment:
Not sure this function is needed as the wrapper should be used. Where is this used?

Reply:
Good catch :handshake: Removed it!